src/app/seo.service.ts

```diff
@@ -25,6 +25,9 @@ export class SeoService {
     this.metaService.updateTag({ name: 'twitter:title', content: title });
     this.metaService.updateTag({ name: 'twitter:description', content: description });
     this.metaService.updateTag({ name: 'twitter:image', content: image });
-  }
 
+    if (prerender_status_code != '200') {
```

Comment:
```suggestion
    if (prerender_status_code !== '200') {
```
it could be a strong check here, just as a good practice :)

Reply:
Computers are a joke.